### ISSUE TITLE
cmake -B build Error: Unknown CMake command "target_link_to_dependencies"

### ISSUE THREAD

**Issue by GeoLibra** [BabylonJS/BabylonNativeExamples#4]
Hi, When I followed the instructions in the README, I encountered this error. 
```bash
1. Run `npm install` from this folder.
2. Run `cmake -B Build` from the repo root.
```
![image](https://github.com/BabylonJS/BabylonNativeExamples/assets/17905890/29bfdb23-70b6-4302-b29e-50287311cfcf)
Could you please help me resolve it? Thanks

**Comment by bghgary** [comment]
I think maybe you ran the cmake command from the `Apps` folder instead of the repo root?

**Comment by GeoLibra** [comment]
![image](https://github.com/BabylonJS/BabylonNativeExamples/assets/17905890/17f1b03e-f437-4825-b47f-b5dd0aa4fae8)
Thank you for your reply, but I tried it and still got the same error

**Comment by bghgary** [comment]
It still looks like from your screenshot that you are running cmake from the `Apps` folder? Try running cmake from the `E:\program\BabylonNativeExamples` folder (the root folder of the repo).

**Comment by bghgary** [comment]
I also just merged [a PR](#5) that cleans some things up.

**Comment by GeoLibra** [comment]
I finally made it，

> running cmake from the E:\program\BabylonNativeExamples folder (the root folder of the repo)

 Thanks 
 
![image](https://github.com/BabylonJS/BabylonNativeExamples/assets/17905890/b3ad3d21-e88a-443c-b873-7958d4dac925)
